resources/views/bread/edit-add.blade.php

```diff
@@ -166,6 +189,11 @@ class="form-edit-add"
                 $('#confirm_delete_modal').modal('show');
             });
 
+            $('.js-delete').on('click', function (e) {
+                $('#delete_form')[0].action = '{{ route('voyager.'.$dataType->slug.'.destroy', ['id' => '__id']) }}'.replace('__id', $(this).data('id'));
```

Comment:
I think I'd rather see this use `.get(0)` instead of `[0]`.  I realize this code was likely copied from somewhere, but if you could update that, I'd appreciate it.